### PR TITLE
feat: Accelerometer Limited Axes (Uncalibrated) sensor Implementation

### PR DESCRIPTION
**Pull request by aditya-bhawsar-mm**
This PR adds support for the [proximity sensor](https://developer.android.com/reference/android/hardware/SensorEvent#sensor.type_accelerometer_limited_axes_uncalibrated:)

README is also updated to include the abovesaid implementation's composable
(`rememberAccelerometerLimitedAxesUncalibratedSensorState()`)

### CODE REVIEW

File: composesensors/src/main/java/com/mutualmobile/composesensors/AccelerometerLimitedAxesUncalibratedSensorState.kt [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ * @param yBias Estimated Y bias (if supported).
+ * @param zBias Estimated Z bias (if supported).
+ * @param xAxisSupported  Acceleration supported for x-axis. Defaults to false.
+ * @param yAxisSupported Acceleration supported for y-axis. Defaults to false.
+ * @param zAxisSupported Acceleration supported for z-axis. Defaults to false.
+ * @param isAvailable Whether the current device has an accelerometer limited axes (uncalibrated) sensor. Defaults to false.
+ * @param accuracy Accuracy factor of the accelerometer limited axes (uncalibrated) sensor. Defaults to 0.
+ */
+@Immutable
+class AccelerometerLimitedAxesUncalibratedSensorState internal constructor(
```
Comment (shubhamsinghmutualmobile) [inline composesensors/src/main/java/com/mutualmobile/composesensors/AccelerometerLimitedAxesUncalibratedSensorState.kt:25]: @aditya-bhawsar-mm Let's please rename the class' and the composable function's name to match the [existing code](https://github.com/mutualmobile/ComposeSensors/blob/main/composesensors/src/main/java/com/mutualmobile/composesensors/UncalibratedMagneticFieldSensorState.kt) i.e. `UncalibratedLimitedAxesAccelerometerSensorState` and `rememberUncalibratedLimitedAxesAccelerometerSensorState`

File: composesensors/src/main/java/com/mutualmobile/composesensors/AccelerometerLimitedAxesUncalibratedSensorState.kt [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+
+/**
+ * An acceleration sensor determines the acceleration that is applied to a device by measuring the
+ * forces that are applied to the sensor itself.
+ * @param xForce Uncalibrated X force without bias compensation (if supported).
+ * @param yForce Uncalibrated Y force without bias compensation (if supported).
+ * @param zForce Uncalibrated Z force without bias compensation (if supported).
```
Comment (shubhamsinghmutualmobile) [inline composesensors/src/main/java/com/mutualmobile/composesensors/AccelerometerLimitedAxesUncalibratedSensorState.kt:14]: Let's also add the `Defaults to 0f` statement here

Comment (aditya-bhawsar-mm) [reply composesensors/src/main/java/com/mutualmobile/composesensors/AccelerometerLimitedAxesUncalibratedSensorState.kt]: done

File: composesensors/src/main/java/com/mutualmobile/composesensors/AccelerometerLimitedAxesUncalibratedSensorState.kt (hunk 1 above)
Comment (aditya-bhawsar-mm) [reply composesensors/src/main/java/com/mutualmobile/composesensors/AccelerometerLimitedAxesUncalibratedSensorState.kt]: done

**Review by shubhamsinghmutualmobile — APPROVED** [review]

**Pull request merged**